rxlifecycle-kotlin/build.gradle

```diff
@@ -13,11 +13,13 @@ android {
 
 repositories {
     mavenCentral()
+    maven { url 'https://maven.google.com' }
 }
 
 dependencies {
     compile kotlinStdlib
     compile project(':rxlifecycle-android')
+    provided project(':rxlifecycle-android-lifecycle')
```

Comment:
Although laborious, I think that's the right way to go. Otherwise people end up pulling in more than they bargained for when they pull in this project.